src/pages/publish-LDS-page/validate-publish-LDS-page.ts

```diff
@@ -1,11 +1,11 @@
 import Joi, { ValidationErrorItem } from "joi";
 
 import { TimetableEventKey } from "@lib/constants";
-import { eventDateSchema } from "../event-schema";
+import { startDateSchema } from "../stage-schema";
 import { ValidateFormParams } from "../FormPageHoc";
 
 const schema = Joi.object({
```

Comment:
for consistency with the other schemas (i.e. statusChangeEventSchema) this should be named something like publishLDSEventSchema